Table

biumiamy opened the pull request.

biumiamy committed:
table

- test table with list item
- and one column less

biumiamy committed:
update

- add tags

biumiamy committed:
update

- change table tags

biumiamy committed:
update

- create output of dataframe using pandas

eliselavy commented on article.ipynb:
```diff
@@ -3,11 +3,6 @@
   {
```
<p>@biumiamy you can remove the markdown table</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/jdh-observer/8X2RX68Qavxr/pull/2/discussion/#comment-1751896822'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

eliselavy replied:
<p>@biumiamy display the dataframe from the csv created</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/jdh-observer/8X2RX68Qavxr/pull/2/discussion/#comment-1751896824'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

biumiamy committed:
update

- delete markdown table
- export as csv
- test: create as dataframe from scratch
- tag

biumiamy committed:
update

- update more lists

biumiamy committed:
update

- change the text more

biumiamy committed:
update

- add all rows

biumiamy committed:
update

- add all data

biumiamy committed:
update

- format

biumiamy committed:
update

formatting

biumiamy committed:
update

update metadata and tag
